etc/hub.api.md

```diff
@@ -48,10 +64,12 @@ export { ArchiveStatusReply }
 
 export { ArchiveWatchReply }
 
+// Warning: (ae-incompatible-release-tags) The symbol "Buckets" is marked as @public, but its signature references "GrpcAuthentication" which is marked as @internal

```

Comment:
Is this an issue?

Reply:
I hid it intentionally so it doesn't make the docs unclear. This warning just means that where docs reference it now it wont be linked to anything. So in particular, when it says Users class is an instance of this, it's just plaintext